### PR TITLE
TRUNK-5548: Upgrade org.powermock Libraries

### PR DESCRIPTION
**Pull request by ODORA0**
**Issue worked on:** https://issues.openmrs.org/browse/TRUNK-5548

### CODE REVIEW

**Comment by ODORA0** [comment]
@mozzy11 This locally passes though, not sure why tests would fail here

**Comment by mozzy11** [comment]
@ODORA0 are you sure it passes locally?

**Comment by ODORA0** [comment]
Yes, I actually run `mvn clean install` twice before this push. l could re-run the command and confirm again but it did build successfully

**Comment by ODORA0** [comment]
@mozzy11 @Ruhanga  The build passes locally still but Travis fails here so I still don't get this, anything missing here?

**Comment by Ruhanga** [comment]
@ODORA0, have you tried some google search on the cause?

**Comment by HerbertYiga** [comment]
@ODORA0   can you try pushing this again!!!

**Comment by ODORA0** [comment]
@mozzy11 right, @Ruhanga  been comparing versions dependencies as well making changes to code but some errors arising as well although some of the dependencies were in beta versions I guess causing that

**Comment by ODORA0** [comment]
Hey @Ruhanga  @mozzy11 I still get this build fail `The POM for junit:junit:jar:5.4.2 is missing, no dependency information available` when i update the Junit:Junit dependency, according to [THIS](https://mvnrepository.com/artifact/org.powermock/powermock-module-junit4/2.0.2) is the latest version. Although, I have been able to update including the other dependencies for these two libraries. 

I get this [Patebin error](https://pastebin.com/1QNMyMAm) if I restore that change to it's original `4.12` I need some guidance on this one. Not sure here!!

**Comment by ODORA0** [comment]
@HerbertYiga  I think that push hasn't done much in changing the Travis report

**Comment by gitcliff** [comment]
@mozzy11  couldn't this failure be the same issue @isears  was addressing recently on talk

**Comment by mozzy11** [comment]
No @gitcliff , this is completely different. What @isears  was addresing was nt cause by a local change.

try adding add an explicit dependency  of powermock-core:
like 

<dependency>
    <groupId>org.powermock</groupId>
    <artifactId>powermock-core</artifactId>
    <version>2.0.2</version>
    <scope>test</scope>
</dependency>

**Comment by ODORA0** [comment]
@dkayiwa I changed the version to 1.7.0 since most of the later ones caused Travis failures. Also added an explicit dependency, is it merge-able now?

**Comment by dkayiwa** [comment]
Did you say that on the ticket?

File: pom.xml [hunk 1]
```diff
@@ -466,15 +466,21 @@
 				<artifactId>mockito-core</artifactId>
 				<version>1.10.19</version>
 			</dependency>
+			<dependency>
+				<groupId>org.powermock</groupId>
```
Comment (dkayiwa) [inline pom.xml]: What happens when you do not add in this new dependency?

Comment (ODORA0) [reply pom.xml]: @dkayiwa I still have a successful build locally

Comment (dkayiwa) [reply pom.xml]: So why did you add the dependency?

Comment (ODORA0) [reply pom.xml]: All previous builds failed and adding the dependency caused less errors, let me update the PR then

**Comment by ODORA0** [comment]
Removed the dependency @dkayiwa

**Pull request merged**